### PR TITLE
LPS-64600 After upgrading from 6.1/6.2 to 7.0, com.liferay.contacts.web stays at version 0.0.0

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-64600](https://issues.liferay.com/browse/LPS-64600).<br><br>

### CODE REVIEW

File: modules/apps/foundation/contacts/contacts-web/src/main/java/com/liferay/contacts/web/upgrade/ContactsWebUpgrade.java [hunk 1]
```diff
@@ -28,6 +29,10 @@
 	@Override
 	public void register(Registry registry) {
 		registry.register(
+			"com.liferay.contacts.web", "0.0.0", "1.0.0",
+			new DummyUpgradeStep());
```
Comment (JorgeFerrer) [inline modules/apps/foundation/contacts/contacts-web/src/main/java/com/liferay/contacts/web/upgrade/ContactsWebUpgrade.java]: @migue @csierra I'm not familiar enough with the new upgrade fw to judge this. Does it look ok to you?

Drew, if it looks good to them, go ahead and send it to Brian.

Comment (csierra) [inline modules/apps/foundation/contacts/contacts-web/src/main/java/com/liferay/contacts/web/upgrade/ContactsWebUpgrade.java]: it seems OK to me

**Comment by JorgeFerrer** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by JorgeFerrer** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/38323. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**